packages/core/builders/CalendarEvent/class.ts

```diff
@@ -16,7 +16,7 @@ class CalendarEventClass implements CalendarEvent {
   organizer!: Person;
   attendees!: Person[];
   description?: string | null;
-  team?: { name: string; members: string[] };
+  team?: { name: string; members: Person[] };
```

Comment:
Person has a lot of other props that we don't provide to `team.members`, so I think we should create another type for it. Maybe `TeamMember`. I quickly did it and it told me about a dead code branch.

We are accessing `member.id `which doesn't exist and thus member.name would be used always

Reply:
https://github.com/calcom/cal.com/pull/7207/commits/5288feca7a182d3e1779124ed58e69e8a2151e68